pkg/recipes/terraform/config/providers/aws.go

```diff
@@ -43,6 +44,16 @@ const (
 	awsRegionParam    = "region"
 	awsAccessKeyParam = "access_key"
 	awsSecretKeyParam = "secret_key"
+
```

Comment:
do we want to add "radiusucp" , "radius-terraform-" etc as consts here?

Reply:
added